controllers/suite_test.go

```diff
@@ -86,6 +87,8 @@ func createOperatorNamespace(ramenNamespace string) {
 }
 
 var _ = BeforeSuite(func() {
+	controllers.ControllerType = ramendrv1alpha1.DRHub
```

Comment:
This is an interesting observation, as we just run the controllers and not setup any other config map, the envtests were/are working.